frontend/packages/dev-console/src/components/pipelineruns/TaskStatusToolTip.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { TaskStatus, mapRunStatusColor } from '../../utils/pipeline-augment';
+
+interface TaskStatusToolTipProps {
+  taskStatus: TaskStatus;
+}
+
+const getMessage = (status: string, value: number = 0): string => {
+  if (status === 'Notstarted' || status === 'FailedToStart') {
```

Comment:
We should look at creating an enum for the known status values. Not sure if existing pipelines code has these strings. If so we can do a separate PR to track the refactor.

Reply:
No, we don't have strings for these values, it's better to have a common enum